Docker-files/Dockerfile-dev

```diff
@@ -0,0 +1,55 @@
+# Use Ubuntu 22.04 as the base image
+FROM ghcr.io/deepmodeling/deepmd-kit:devel_cu11
```

Comment:
@rebeltroopers what is `devel_cu11`? I can't understand it from the[repo](https://github.com/deepmodeling/deepmd-kit/pkgs/container/deepmd-kit/190590505?tag=devel_cu11).

Reply:
devel_cu11 uses CUDA11 as a base image. The devel is for development version, as you need it to compile with nvcc. So it would be deepmd using cuda 11 with all development tools as well.
https://hub.docker.com/r/nvidia/cuda